main.c

```diff
@@ -14,8 +14,13 @@
   #include <SDL/SDL_mixer.h>
 #else
   #include "SDL_ttf.h"
+	#if defined(NXDK)
```

Comment:
This comes unexpected where it is and only affects non-emscripten yet.
This should be at the very top, before any includes.